OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -26,15 +26,15 @@ public class CombatDebugOverlay : IPostRender, INotifyDamage
 	{
 		readonly DeveloperMode devMode;
 
+		readonly HealthInfo health;
 		Lazy<AttackBase> attack;
 		Lazy<IBodyOrientation> coords;
-		Lazy<Health> health;
```

Comment:
Is removing the laziness here safe without some sort of ordering guarantee of the trait construction order?